domain/src/main/java/com/sitionix/athssox/domain/User.java

```diff
@@ -0,0 +1,19 @@
+package com.sitionix.athssox.domain;
+
+import lombok.*;
+import org.springframework.stereotype.Component;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Component
```

Comment:
why do you need this annotation here @Component

Reply:
Remove annot